.coderabbit.yml

```
@@ -1,9 +1,87 @@
+docstrings:
+  enabled: true
+unit_tests:
+  enabled: true
+
+pre_merge_checks:
+title:
+  mode: "error"
+  requirements: "Title should be concise and descriptive"
+description:
+  mode: "warning"
+docstrings:
```

YAMLlint (1.37.1)

[error] duplication of key "docstrings" in mapping

(key-duplicates)